### PR TITLE
create model's and crud api for that model

### PR DESCRIPTION
**Pull request by pydev-rahul**
in api i've get data of user and sending email based on condition.

### CODE REVIEW

File: sum.py [hunk 1]
```diff
@@ -20,7 +20,8 @@
 
 #print the minimum keys
 print("".join(minimum_keys))
-
+print("".join(minimum_keys))
```
Comment (pydev-rahul) [inline sum.py:23]: hey mistakenly you write print statement two times 
please remove that logs. 
thanks.

Comment (pydev-rahul) [reply sum.py:23]: removed print statement thanks for the review

File: sum.py [hunk 2]
```diff
@@ -20,7 +20,8 @@
 
 #print the minimum keys
 print("".join(minimum_keys))
-
```
Comment (HardikLogicrays) [inline sum.py:23]: remove this unused line

Comment (pydev-rahul) [reply sum.py:23]: removed.

**Pull request merged**